judo-requirement-report-test/src/test/java/hu/blackbelt/judo/requirement/test/TestRequirementProcessor.java

```diff
@@ -210,15 +233,19 @@ public Set<? extends Element> getElementsAnnotatedWith(Class<? extends Annotatio
 
     @Test
     /*
-     * This is the real test case. This case checks the generated csv.
+     * This is a real test case. This case checks the generated requirements-report.csv.
      */
-    public void testReal02(){
+    public void testReal02() {
         String reportPath = System.getProperty("reportPath");
         // The value of this property has to be the same as reportPath compiler argument.
         assertNotNull(reportPath, "The reportPath system variable must be set.");
-
+        
+        String fileNameOfReqReportCsv = (reportPath.endsWith("/")) ?
+                reportPath + RequirementProcessor.REQUIREMENT_REPORT_CSV
+                : reportPath + "/" + RequirementProcessor.REQUIREMENT_REPORT_CSV;
```

Comment:
I'd use parentheses because this code could be ambiguous:
```
reportPath.endsWith("/") ? reportPath + RequirementProcessor.REQUIREMENT_REPORT_CSV 
                         : (reportPath + "/" + RequirementProcessor.REQUIREMENT_REPORT_CSV)
```
```
reportPath.endsWith("/") ? reportPath + RequirementProcessor.REQUIREMENT_REPORT_CSV 
                         : (reportPath + "/")
    + RequirementProcessor.REQUIREMENT_REPORT_CSV
```
```
reportPath.endsWith("/") ? reportPath + RequirementProcessor.REQUIREMENT_REPORT_CSV 
                         : reportPath
     + "/" + RequirementProcessor.REQUIREMENT_REPORT_CSV
```

I might be mistaken, but have seen bugs originating from this use of ternary operator with strings concatenations

Reply:
If tests were successful and their result was checked against this path building logic, this comment can be considered as resolved

Reply:
I have tested and it works how I tought. But I am using the paretheses.